### PR TITLE
[AB] [83044208] Describe refresh token usage

### PR DESCRIPTION
**Pull request by kaborso**
Give an overview of how one might implement a client which manages a session with refresh tokens.

### CODE REVIEW

File: client/oauth2_client.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   site:          OAUTH2_PROVIDER,
   authorize_url: OAUTH2_AUTHORIZE_URL,
   token_url:     OAUTH2_ACCESS_TOKEN_URL }
-
+TIMEOUT_WARNING         = 60
```
Comment (Ebtoulson) [inline client/oauth2_client.rb]: I'm assuming this is seconds? Not sure if a one minute warning is long enough. What if someone is on a page for 2+ minutes and during that time the token expired? Is that acceptable or should it be something like 5 minutes?

Comment (kaborso) [inline client/oauth2_client.rb]: I got rid of this altogether.

**Comment by kaborso** [comment]
@Ebtoulson :clap:

**Comment by Ebtoulson** [comment]
lol get out of here @kaborso . That stuff was actually broken for me hence [this commit](https://github.com/rentpath/portal_integration_samples/commit/bea23d25b8951f1e1f50d0b0365dcf9fdddc8750)

**Comment by colinrymer** [comment]
:clap: :tada:

**Comment by Ebtoulson** [comment]
LMAO ex-rentpath employees party

**Pull request merged**